packages/core/src/constants/type/Config.ts

```diff
@@ -12,7 +12,8 @@ export type FullConfig = {
   readOnlyChainId?: ChainId
   readOnlyUrls?: NodeUrls
   multicallAddresses?: MulticallAddresses
-  supportedChains: number[]
+  supportedChains?: number[]
+  networks?: Chain[]
```

Comment:
Looks good, I would add in [useConfig](https://github.com/EthWorks/useDApp/blob/cdb02b9a260b99d85622401a83037bcea07c8a02/packages/core/src/providers/config/context.ts#L10) sth like:
```ts
if (config.supportedChains) {
  const networks = getNetworksByIds(config.supportedChains)
  return {...config, networks}
}   
return config
```

So that we still use supportedNetworks if someone is using it while fallback to networks if no supportedChains are present.